### PR TITLE
Java: add Apache Ant

### PR DESCRIPTION
**Pull request by gesellix**
This patch adds support to install Apache Ant via SDKMAN!, see https://sdkman.io/sdks#ant

I haven't found detailed documentation about contributing such a feature request. Please leave some feedback if Apache Ant is completely out of scope for the Java feature. Maybe you have some thoughts about more generic support for the SDKMAN! supported tools.

The tests have been adjusted to include Ant, but I haven't run them locally, yet (hoping for the CI to do that for me 😸 ).

### CODE REVIEW

**Comment by gesellix** [comment]
@microsoft-github-policy-service agree

File: src/java/README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Installs Java, SDKMAN! (if not installed), and needed dependencies.
 | gradleVersion | Select or enter a Gradle version | string | latest |
 | installMaven | Install Maven, a management tool for Java | boolean | false |
 | mavenVersion | Select or enter a Maven version | string | latest |
+| installAnt | Install Ant, a software tool for automating software build processes | boolean | false |
```
Comment (samruddhikhandale) [inline src/java/README.md:24]: Thanks for being considerate, however, This file is autogenerated with https://github.com/devcontainers/action which utilizes the `options`. Can we revert this change to avoid merge conflicts?

File: src/java/devcontainer-feature.json [hunk 2]
```diff
@@ -59,6 +59,21 @@
       ],
       "default": "latest",
       "description": "Select or enter a Maven version"
```
Comment (samruddhikhandale) [inline src/java/devcontainer-feature.json:61]: Can we update the minor version in here?
https://github.com/devcontainers/features/blob/21b034ed57a0b3c09ddd74e0995fe884c7566a28/src/java/devcontainer-feature.json#L3

File: src/java/devcontainer-feature.json [hunk 3]
```diff
@@ -59,6 +59,21 @@
...
+    },
+    "antVersion": {
+      "type": "string",
+      "proposals": [
+        "latest",
+        "1.10.12",
+        "1.9.15"
+      ],
+      "default": "latest",
+      "description": "Select or enter a Ant version"
```
Comment (samruddhikhandale) [inline src/java/devcontainer-feature.json:76]:
```suggestion
      "description": "Select or enter an Ant version"
```

File: test/java/scenarios.json [hunk 4]
```diff
@@ -26,26 +26,30 @@
             },
             "java": {
                 "version": "latest",
+                "installAnt": true,
```
Comment (samruddhikhandale) [inline test/java/scenarios.json:29]: Can we add tests to validate this? (in https://github.com/devcontainers/features/blob/21b034ed57a0b3c09ddd74e0995fe884c7566a28/test/java/install_ant_and_gradle_and_maven.sh & others? )

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks for adding another Feature option, seems pretty useful to me. 
Left few comments!

**Comment by gesellix** [comment]
Thanks for your quick feedback!

The changes have been added like you requested, but I failed to keep the version bump in a seperate commit. I hope you plan to squash the commits, so this probably won't be an issue? If you prefer to keep dedicated commits, I can restructure the commits and make a forced push.

**Review by samruddhikhandale — APPROVED** [review]
Nice, LGTM ⚡ 

ℹ️ The test failure for `install_from_non_default_distro` also happens on `main` branch & is unrelated to this PR.

**Comment by samruddhikhandale** [comment]
>The changes have been added like you requested, but I failed to keep the version bump in a seperate commit. I hope you plan to squash the commits, so this probably won't be an issue? If you prefer to keep dedicated commits, I can restructure the commits and make a forced push.

That's fine. The PR merges with a squash commit

**Comment by gesellix** [comment]
Thanks a lot for the quick feedback and merge!
